cellxgene_schema_cli/tests/questions_workshop.md

```diff
@@ -0,0 +1 @@
+1. 
```

Comment:
What is this?

Reply:
hm I don't know what ended up there, I'll delete it